admin-site/src/Login.svelte

```diff
@@ -1,27 +1,27 @@
 <!-- Login component with a simple password verification -->
 
  <script>
-    export let authenticated = false;
+  import Page from './ParentPage.svelte';
+  import Button from './Button.svelte';
+  
+  export let authenticated = false;
+  export let serverURL;
 
     function verifyPassword(){
```

Comment:
This seems like the logical place for this function... why is the same function (almost) also in App.svelte?

If you need `App` to do something when `authenticated = true`, my first thought would be to have `Login` emit an event that is consumed by `App`. https://svelte.dev/tutorial/component-events

Reply:
Login isn't implemented as a component yet. I started then stopped to check in. I think that's a good solution though. I'll comment here again once it's included.